TKeazirian.HTTPServer.Tests/helpers/HelperFunctions.cs

```diff
@@ -20,4 +22,10 @@ public static string CreateTestResponseHeaders()
 
         return testResponseHeaders;
     }
+
+    public static Request ImproperFormattedRequest(string verb, string path, string body)
+    {
+        Request testRequest = new Request(body, verb, CreateTestResponseHeaders(), path);
+        return testRequest;
+    }
```

Comment:
typically helper functions are for code that will be frequently reused (generic cases).  This helper is very specific, and therefore the code may serve the test it is used in better, to be inlined there.